l10n_de_tax_statement/README.rst

```diff
@@ -0,0 +1,131 @@
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--germany-lightgray.png?logo=github
+    :target: https://github.com/OCA/l10n-germany/tree/12.0/l10n_de_tax_statement
+    :alt: OCA/l10n-germany
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/l10n-germany-12-0/l10n-germany-12-0-l10n_de_tax_statement
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/175/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module provides the  *German VAT Statement* (Umsatzsteuervoranmeldung).
+You can use the *German VAT Statement* report to declare your taxes on www.elster.de.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Install module *l10n_de_tax_statement* version >= *12.0.1.0.0*.
+#. This module depends on module account_tax_balance available at https://github.com/OCA/account-financial-reporting.
+
+Configuration
+=============
+
+
+This module makes use of the tax tags (eg.: 81, 86, 91, 89, 93 ...) as prescribed by the German tax laws and represented by the tax declaration form "Umsatzsteuervoranmeldung".
+
+If the default Odoo German chart of accounts is installed (module l10n_de) you are able to select in the settings if you want to use the skr03 or skr04 chart variant. By installing and configuring your favored german account chart the tax tags from the module l10n_de are automatically present in the database. If this is the case, go to menu: Invoicing -> Configuration -> Accounting -> German Tax Tags, and check that the tags are correctly set. Click finally Apply to confirm the right tax - tax tags mappings.
+
+If a non-standard chart of accounts is installed, you have to manually create the tax tags and properly set them into the tax definition. If you create another german account chart (f.e. l10n_de_ikr) you can still depend on l10n_de module in order to benefit from the generic tax tags for germany. If you won't use l10n_de as a base module you have to configure at first your own tax tags. After that, go to go to menu: Invoicing -> Configuration -> Accounting -> German Tax Tags, and manually set the tags in the configuration form; click Apply to confirm (for more information about the installation and configuration of that module, check the README file).
+
+The user must belong to the *Show Full Accounting Features* group, to be able to access the `Invoicing -> Configuration -> Accounting -> NL BTW Tags` menu.
```

Comment:
Should be "DE" for German localization?

Reply:
Yes, but in this case the name of the menu is: `Ust-Voranmeldung`. Fixed!

Reply:
:thinking: but in other places of the readme file there's written `German Tax Tags` for the same menu. I'm going to set `German Tax Tags` in the menu and update the readme accordingly.

Reply:
Yeah, menu names needs to be the same in readme as in odoo.  :+1:

Reply:
Ok, should be correct now:
`Invoicing -> Configuration -> Accounting -> German Tax Tags`.